lib/globus/client/identity.rb

```diff
@@ -18,6 +18,13 @@ def get_identity_id(sunetid)
         extract_id(data)
       end
 
+      def exists?(sunetid)
+        get_identity_id(sunetid)
+        true
+      rescue
+        false
+      end
```

Comment:
Can you extend the identity spec to make sure these lines are covered?

Reply:
yes, added new tests